### DISCUSSION TITLE
Mapped types vs column types

### DISCUSSION THREAD

**Discussion by johnnyomair** [mikro-orm/mikro-orm#6261] [General]
I've been using MikroORM for quite some time now and been wondering if there is a recommendation on which types to prefer, the mapped types (`type`) or column types (`columnType`) for types that are supported by both (e.g., `text`) . It seems to me that mapped types a preferred (since they're used first in the [tutorial](https://mikro-orm.io/docs/guide/first-entity#scalar-properties)). Could you please confirm this? Is this something we could document or even lint?

**Comment by B4nan** [comment]
Correct, `type` option should be preferred, `columnType` is there so you can override what gets inferred.

**Reply by johnnyomair** [reply]
Thanks!
